.github/workflows/devbuild.yml

```diff
@@ -249,15 +261,13 @@ jobs:
       - name: event name
         run: |
           echo "github.event_name: ${{ github.event_name }}"
+          # Some mac runner does not have /usr/local/include and cmake sometimes crashes
```

Comment:
At the time being, macos-14 image does not have the directory `/usr/local/include`.  It is not clear whether macos-13 image has it.  Create the directory regardless.

If the directory is not there, `make gtest` on macos-14 fails with the error messages saying cmake cannot find that directory.